exe/task_wrapper.cc

```diff
@@ -44,7 +45,9 @@ int main(int argc, char *argv[])
         // Avoid atomic update to allow testing against /dev/stderr. There should never be
         // multiple processes trying to write this output.
         boost::nowide::ofstream ofs { params.get<std::string>("stderr"), std::ios::binary };
+        #ifndef _WIN32
         fs::permissions(params.get<std::string>("stderr"), FILE_PERMS);
```

Comment:
And stylistically, we tend to put the pragmas at the beginning of the line (not inline with code).